site-src/implementations.md

```diff
@@ -0,0 +1,95 @@
+# Implementations
+
+This document tracks downstream implementations of Gateway API and provides status and resource references for them.
+
+Implementors of Gateway API are encouraged to update this document with status information about their implementations, the versions they cover, and documentation to help users get started.
+
+## Implementation Status
+
+|                | [Contour][1]       | [GCP][2]           | [Istio][3]         | [Kong][4]         | [Knative][5]       | [Traefik][6]       |
+| :------------- | :----------------: | :----------------: | :----------------: | :---------------: | :----------------: | :----------------: |
+| [v0.1.x][v0.1] | :heavy_minus_sign: | :heavy_minus_sign: | :heavy_check_mark: |:heavy_minus_sign: | :heavy_minus_sign: | :heavy_check_mark: |
+| [v0.2.x][v0.2] | :heavy_check_mark: | :heavy_plus_sign:  | :heavy_plus_sign:  |:heavy_plus_sign:  | :heavy_minus_sign: | :heavy_plus_sign:  |
+
+- :heavy_check_mark: available
+- :heavy_plus_sign: in progress
+- :heavy_minus_sign: not planned
+
+[1]:#contour
+[2]:#google-cloud-platform
+[3]:#istio
+[4]:#kong
+[5]:#knative
+[6]:#traefik
+
+[v0.1]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.1.0
+[v0.2]:https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.2.0
+
+## Project References
+
+In this section you will find specific links to blog posts, documentation and other Gateway API references for specific implementations.
+
+### Contour
+
+[Contour][contour] is an open source ingress controller for Kubernetes.
+
+Contour currently implements the latest `v0.2.x` Gateway API Specification.
+
+See the [Contour Gateway API Guide][contour-1] for information on how to deploy and use Contours's Gateway implementation.
+
+[contour]:https://projectcontour.io
+[latest]:https://doc.traefik.io/traefik/routing/providers/kubernetes-gateway/
+[contour-1]:https://projectcontour.io/guides/gateway-api/
+
+### Google Cloud Platform
+
+The [Google Cloud Platform (GCP)][gcp] is a cloud computing platform and infrastructure provider.
+
+GCP is actively working towards an implementation of the Gateway API `v0.2.x` specification, and status updates and documentation will be provided here as the work progresses.
+
+[gcp]:https://cloud.google.com
+
+### Istio
+
+[Istio][istio] is an open source [service mesh][mesh] for Kubernetes clusters.
+
+Istio currently supports the previous `v0.1.x` Gateway API specification and is working towards supporting the latest `v0.2.x` release.
+
+Status updates and documentation for `v0.2.x` will be provided here as the work progresses.
```

Comment:
Need to verify this, as I was just guessing when I made the PR draft, I'll do some more digging before I take this out of draft and make it ready for review.

Reply:
we are importing the v0.2.x code now, but its in our `master` branch not a release yet and our docs aren't updated yet to point to the new "Gateway" name. Not sure what emoji that turns into :slightly_smiling_face:

Reply:
Understood, thanks for the heads up! I'll try to get this updated to better reflect the state of things.

Reply:
I've updated this to be simpler and then more context can be added later in future iterations, lmkwyt :+1: 

5f51da790e097b5030548838e5bd51abceadc3fb